config/environments/production.rb

```diff
@@ -53,16 +47,19 @@
   config.log_level = :debug
 
   # Prepend all log lines with the following tags.
-  # config.log_tags = [ :subdomain, :uuid ]
-
-  # Use a different logger for distributed setups.
-  # config.logger = ActiveSupport::TaggedLogging.new(SyslogLogger.new)
+  # config.log_tags = [ :request_id ]
```

Comment:
It was [already commented out](https://github.com/alphagov/content-tagger/pull/345/files#diff-1d98f0039bbc6df92adee882ed99d993L56)